src/Coff/Object.zig

```diff
@@ -145,44 +104,39 @@ fn parseShdrs(self: *Object, allocator: Allocator) !void {
 
 fn parseSymtab(self: *Object, allocator: Allocator) !void {
     const offset = self.header.pointer_to_symbol_table;
+    if (offset == 0) return error.NoSymtab;
```

Comment:
I don't think this should be an error. For ELF/MachO it is perfectly fine though uncommon to receive an object file with no symbols. In that case, we should simply record that there are no symbols and move on. Also, strtab should really be paired with parsing of symtab unless it contains strings that cover more than just the symtab.

Reply:
Section Table also uses strings defined in strtab for names longer than 8 characters in object files.